PlusMinus.hs

```diff
@@ -0,0 +1,7 @@
+fraction func array = show $ (fromIntegral $ func array) / n
```

Comment:
Syntactically I don't think it's bad, but there's a lot of `$` and parentheses. It's a symptom of this function doing too much really. It's ok to have a `doTheThing` function at the top level, but it's important that it's expressive. Here you have some boilerplate, like dividing `Int`s, that could be extracted. Also why is `show` there? I would expect to see it near a `putStrLn` instead. Also, unless the function is trivial, I would add a type annotation. It's not obvious that `func array` should return a `Int`. I think if you make the function do only one thing, the `$`s should go away. You probably shouldn't have more than one of those in a line anyway. Maybe there are some haskell style guides? Linters?